tests/unit/scalyr_logging_test.py

```diff
@@ -72,6 +73,63 @@ def test_formatter(self):
         )
         self.assertLogFileContainsLineRegex(expression=expression)
 
+    def test_debug_level_for_subset_of_loggers(self):
+        logger1 = scalyr_logging.getLogger("logger1")
+        logger2 = scalyr_logging.getLogger("logger2")
+        logger3 = scalyr_logging.getLogger("logger3")
+
+        logger1.info("XX Test info 1")
+        logger2.debug("XX Test debug 1")
+
+        logger2.info("XX Test info 2")
+        logger2.debug("XX Test debug 2")
+
+        logger3.info("XX Test info 3")
+        logger3.debug("XX Test debug 3")
+
+        # 1. Default behavior - only INFO is enabled
+        self.assertLogFileContainsLineRegex(expression="XX Test info 1")
+        self.assertLogFileContainsLineRegex(expression="XX Test info 2")
+        self.assertLogFileContainsLineRegex(expression="XX Test info 3")
+
+        self.assertLogFileDoesntContainsLineRegex(
+            expression="XX Test debug 1", file_path=self.agent_debug_log_path
+        )
+        self.assertLogFileDoesntContainsLineRegex(
+            expression="XX Test debug 2", file_path=self.agent_debug_log_path
+        )
+        self.assertLogFileDoesntContainsLineRegex(
+            expression="XX Test debug 3", file_path=self.agent_debug_log_path
+        )
+
+        # 2. Now we enable debug level for 2 loggers only
+        scalyr_logging.set_log_level(
+            level=logging.INFO, debug_level_logger_names=["logger2", "logger3"]
+        )
+
+        logger1.info("YY Test info 1")
+        logger2.debug("YY Test debug 1")
```

Comment:
Should it be the `logger1` instead of 2?

Reply:
Good catch - thanks, will fix it.